MusicHub/tracks/validators.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.serializers import ValidationError
+
+from ..antivirusProvider.service import AntivirusScan
+from ..main.constants import MAX_FILE_SIZE_IN_MB
+
+
+def validate_track(data):
```

Comment:
def validate_track(data, max_size=None):
   if max_size and  data.size >= max_size:
       raise ValidationError()